tools/c7n_gcp/c7n_gcp/output.py

```diff
@@ -155,10 +155,10 @@ def get_handler(self):
 
         log_group = self.get_log_group()
         project_id = local_session(self.ctx.session_factory).get_default_project()
-        client = LogClient(project_id)
+        client = LogClient(project=project_id)
```

Comment:
The argument need to specify the keyword since there is keyword only arguments:

https://github.com/googleapis/python-logging/blob/36fb4270b3cb04014189c0f296ac57cbba6c9127/google/cloud/logging_v2/client.py#L82

This is were the initial panic came from: #10442